### PR TITLE
FIX: Use standard error for die(), dieWithHelp(), and error messages

### PR DESCRIPTION
**Pull request by inkarkat**
**Before submitting a pull request,** please make sure the following is done:

- [X] Fork [the repository](https://github.com/todotxt/todo.txt-cli) and create your branch from `master`.
- [X] If you've added code that should be tested, add tests!
- [X] Ensure the test suite passes.
- [X] Format your code with [ShellCheck](https://www.shellcheck.net/).
- [X] Include a human-readable description of what the pull request is trying to accomplish.
- [ ] Steps for the reviewer(s) on how they can manually QA the changes.
- [ ] Have a `fixes #XX` reference to the issue that this pull request fixes.

### CODE REVIEW

File: todo.sh [hunk 1]
```diff
@@ -1211,9 +1212,11 @@ case $action in
 		echo "TODO: $item deprioritized."
 	    fi
 	else
-	    echo "TODO: $item is not prioritized."
+	    echo >&2 "TODO: $item is not prioritized."
+	    status=1
```
Comment (chrysle) [inline todo.sh:1216]: Is it necessary here to leave the whole process with failure exit status? Maybe you could test if other `$item`s succeed?

Comment (inkarkat) [reply todo.sh:1216]: For me, failure is the inability to do what I was told, even if I partially succeeded. The user still has the action's output to see which items succeeded and which didn't; the exit status is more of interest from a scripting perspective; the shell doesn't visualize the exit status by default (but I color the following prompt red if there was an error). And from that scripting perspective, it's more interesting whether the whole command succeeded, not whether it got partially done. (And you can always use a loop and invoke the command with each item separately to obtain that information.)

**Review by chrysle — APPROVED** [review]

**Comment by chrysle** [comment]
Could you help me out how to use this PR as a base for mine? I can't find any description concerning this.

**Comment by inkarkat** [comment]
@chrysle There's still a long queue of PRs waiting to be merged, so I don't think it's worth the effort to properly base your changes on mine. Just add the `>&2` to the `die()` function manually, and the completion test should pass again (but other tests that I've adapted in this PR will fail). As long as this PR gets merged before your, everything will be fine ;-)

**Pull request merged**